### DISCUSSION TITLE
how to measure total time

### DISCUSSION THREAD

**Discussion by legolas089** [snl-dakota/dakota#122] [Q&A]
Hello, I'm just learning how to use dakota... and I wonder how to measure total operating time. 
Total wall clock is seen in Console area, but it seems it is the time for total operating including the printing results to console area.
How can I get the time just for iteration?

**Comment by jadamstephens** [comment]
Hi,

Dakota does not report that time. You might have to instrument your driver to print out how long it took from start to finish.
